### PR TITLE
Lab refresh - 4000

### PR DESCRIPTION
**Pull request by aberinnj**
# Add here your WMS ID
4000

# General requirements
1. Information in the workshop is adequate and updated
2. Code is correct and working
3. Links are correct
4. Make sure you enter the help email link in your manifest.json
5. Please make sure WMS URLs are updated as needed after your PR is approved

# Checklist - Refer to the QA document for the complete list 
Please confirm that the following is completed before submitting your PR

- [X] All filenames are lower case (including folders, images, files, etc.)
- [X] Filenames are descriptive
- [X] Your workshop folder structure should be similar to the one used in the sample workshop (https://github.com/oracle-livelabs/common/tree/main/sample-livelabs-templates/sample-workshop)
- [X] Are you using multiple versions (desktop/, sandbox/, tenancy/)? Make sure that each of them contains a manifest.json and an index.html
- [X] Image references in markdown contain an alternate text

### CODE REVIEW

File: mytodoreact/frontend/frontend.md
Comment (denisemyrick) [inline mytodoreact/frontend/frontend.md:1]: Please ensure this isn't jumping from task 2 to task 4.

File: mytodoreact/teardown/teardown.md
Comment (denisemyrick) [inline mytodoreact/teardown/teardown.md:1]: looks like you have two task 4s

**Review by denisemyrick — CHANGES_REQUESTED** [review]

**Comment by aberinnj** [comment]
@denisemyrick Thank you for reviewing so far, please give it another look!

File: mytodoreact/intro/images/Application.png
Comment (denisemyrick) [inline mytodoreact/intro/images/Application.png:1]: You still need to rename Application.png to application.png

**Review by denisemyrick — DISMISSED** [review]

**Review by ramumg — APPROVED** [review]

**Pull request merged**